examples/FCCee/bsm/LLPs/ExoticHiggsDecays/Reco_analysis_stage1.py

```diff
@@ -0,0 +1,264 @@
+            .Define("Reco_mumu_pz", "if ((n_RecoMuons>1) && (RecoMuon_charge.at(0) != RecoMuon_charge.at(1))) return (RecoMuon_pz.at(0) + RecoMuon_pz.at(1)); else return float(-1.);")
+            .Define("Reco_mumu_invMass", "if ((n_RecoMuons>1) && (RecoMuon_charge.at(0) != RecoMuon_charge.at(1))) return sqrt(Reco_mumu_energy*Reco_mumu_energy - Reco_mumu_px*Reco_mumu_px - Reco_mumu_py*Reco_mumu_py - Reco_mumu_pz*Reco_mumu_pz ); else return float(-1.);")
+
+
+            # Jet study
+
+            # define the RP px, py, pz and e
+            .Define("RP_px",          "ReconstructedParticle::get_px(ReconstructedParticles)")
+            .Define("RP_py",          "ReconstructedParticle::get_py(ReconstructedParticles)")
+            .Define("RP_pz",          "ReconstructedParticle::get_pz(ReconstructedParticles)")
+            .Define("RP_e",           "ReconstructedParticle::get_e(ReconstructedParticles)")
+            .Define("RP_m",           "ReconstructedParticle::get_mass(ReconstructedParticles)")
+            .Define("RP_q",           "ReconstructedParticle::get_charge(ReconstructedParticles)")
+
+            # build pseudo jets with the RP, using the interface that takes px,py,pz,e 
+            .Define("pseudo_jets",    "JetClusteringUtils::set_pseudoJets(RP_px, RP_py, RP_pz, RP_e)")
+
+            # run jet clustering with all reconstructed particles, Durham jet-algorithm
+            # ee_kt_algorithm 2: exclusive clustering, exactly njets = 4, 1: sorted by E, 0: recombination scheme E
+            .Define("FCCAnalysesJets_ee_kt", "JetClustering::clustering_ee_kt(2, 4, 1, 0)(pseudo_jets)")
+            #get the jets out of the struct
+            .Define("jets_ee_kt",           "JetClusteringUtils::get_pseudoJets(FCCAnalysesJets_ee_kt)")
+            #get the jets constituents out of the struct
+            .Define("jetconstituents_ee_kt","JetClusteringUtils::get_constituents(FCCAnalysesJets_ee_kt)")
+
+            #get some variables
+            .Define("jets_ee_kt_e",        "JetClusteringUtils::get_e(jets_ee_kt)")
+            .Define("jets_ee_kt_px",        "JetClusteringUtils::get_px(jets_ee_kt)")
+            .Define("jets_ee_kt_py",        "JetClusteringUtils::get_py(jets_ee_kt)")
+            .Define("jets_ee_kt_pz",        "JetClusteringUtils::get_pz(jets_ee_kt)")
+            .Define("jets_ee_kt_m",        "JetClusteringUtils::get_m(jets_ee_kt)")
+
+        )
+        return df2
+
+
+    #__________________________________________________________
+    #Mandatory: output function, please make sure you return the branchlist as a python list
+    def output():
+        branchList = [
+            "n_tracks",
+            "n_RecoedPrimaryTracks",
+            "n_SecondaryTracks",
+            # "SV_evt1_n",
+            # 'invMass_SVs',
+            # "Reco_SVs_Lxy",

```

Comment:
why are these variables and others included but commented out?

Reply:
Aa yes this is left from when I studied the variables of the DVs at different stages of the DV algortihm. Commented out here since running them all takes some time. In a sensitivity analysis though they will not be needed I think, so I will delete them completely from here! But I might add the chi2 distributions of the DVs as well, so I'll do that before updating this PR again with the changes!

Reply:
sounds good!